builds/fabric8-che/assembly/dashboard/src/main/patches/src/app/index.module.ts.patch

```diff
@@ -38,12 +43,12 @@
 +    
 +  }
 +
-+  promise.then((resp) => {
-+    console.log('resp : ' + resp.toString() );
-+    if (resp['che.keycloak.disabled'] == 'true') {
-+      keycloakInit(true);
++  promise.then((keycloakSettings) => {
++    console.log('keycloakSettings : ' + keycloakSettings.toString() );
```

Comment:
I think that using console.log('keycloakSettings : ', keycloakSettings) should work as well but I'm not 100% sure